CODIGO/modNetwork.bas

```diff
@@ -18,19 +18,48 @@ Attribute VB_Name = "modNetwork"
 Option Explicit
 
 Private Client As Network.Client
+
+Private Type t_FailedIp
+    IP As String
+    Port As String
+End Type
+
+Dim FailedIpList(10) As t_FailedIp
+Public FaiedListSize As Integer
+
 #If PYMMO = 0 Then
 Public Function IsConnected() As Boolean
     IsConnected = Connected
 End Function
 #End If
 
+Public Function IsFailedString(ByVal IP As String, ByVal Port As String)
+    Dim i As Integer
+    For i = 0 To UBound(FailedIpList) - 1
+        If FailedIpList(i).IP = IP And FailedIpList(i).Port = Port Then
+            IsFailedString = True
+            Exit Function
+        End If
+    Next i
```

Comment:
Missing  `IsFailedString = False` before leaving the function or at the beginning

Reply:
false is default value